www/class/config-generate/contact.class.php

```diff
@@ -206,11 +207,28 @@ protected function getContactNotificationCommands($contact_id, $label)
         }
     }
 
+    /**
+     * @param integer $contactId
+     * @return boolean
+     */
+    protected function shouldContactBeNotified(int $contactId): bool
+    {
+        if ($this->contacts[$contactId]['enable_notifications'] == '1') {
```

Comment:
Please use a constant instead to be more readable
```suggestion
        if ($this->contacts[$contactId]['enable_notifications'] == self::.....) {
```